pkg/common/common.go

```diff
@@ -167,27 +138,6 @@ func Filter[T any](slice []T, f func(T) bool) []T {
 	return arr
 }
 
-// SliceCopy copies the elements from the input slice into the output slice, and returns the output slice.
-func SliceCopy[T any](s1 []T) []T {
-	s2 := make([]T, len(s1))
-	copy(s2, s1)
-	return s2
-}
-
-// ReverseSlice creates a reversed copy of the input slice.
-func ReverseSlice[T any](input []T) []T {
-	inputLen := len(input)
-	output := make([]T, inputLen)
-
-	for i, n := range input {
-		j := inputLen - i - 1
-
-		output[j] = n
-	}
-
-	return output
-}
-
```

Comment:
Removing as it's already provided by
- https://pkg.go.dev/golang.org/x/exp@v0.0.0-20230817173708-d852ddb80c63/slices#Clone
- https://pkg.go.dev/golang.org/x/exp@v0.0.0-20230817173708-d852ddb80c63/slices#Reverse